sphinx/source/docs/dev_guide/setup.rst

```diff
@@ -84,11 +84,16 @@ At the top level of the repository, issue the following command in a terminal:
 
     conda env create environment.yml
 
-Then, to activate the environment:
+Then, activate the environment:
 
 .. code-block:: sh
 
     conda activate bkdev
+    
+.. note::
+    
+    Before creating a conda environment, initialize Anaconda with this command
+    ``conda init`` only once.
```

Comment:
```suggestion
    Before creating a conda environment for the first time, you may need to 
    initialize Anaconda with the ``conda init`` (only once).
```

Reply:
@afzal442 Let me know what you think off this suggestion. I wanted to make it more clear that this was a "before the first env" not "before any env"

Reply:
Ah, I see. Let me commit your suggestion. Thanks.